===================================================================
RCS file: /var/cvs/Drupal-dev/includes/common.inc,v
retrieving revision 1.1
retrieving revision 1.2
diff -u -r1.1 -r1.2
--- includes/common.inc	2004/09/21 11:16:01	1.1
+++ includes/common.inc	2004/09/22 23:43:31	1.2
@@ -1,5 +1,5 @@
 <?php
-/* $Id$ */
+/* $Id$ */
 
 /**
  * @file
@@ -507,6 +507,9 @@
  */
 function locale_initialize() {
   global $user;
+  
+  if ( function_exists('i18n_locale_init') ) return i18n_locale_init();
+ 
   if (function_exists('locale')) {
     $languages = locale_supported_languages();
     $languages = $languages['name'];